components/search/index.css

```diff
@@ -226,14 +210,26 @@
 
 /* Quiet Variant */
 .spectrum-Search--quiet {
-	.spectrum-Search-clearButton .spectrum-ClearButton-icon {
-		transform: translateX(var(--mod-search-quiet-button-offset, var(--spectrum-search-quiet-button-offset)));
+	--spectrum-search-background-color: transparent;
+	--spectrum-search-background-color-disabled: transparent;
+	--spectrum-search-border-color-disabled: var(--spectrum-disabled-border-color);
+
+	/* Added specificity, otherwise they are overridden by system specific themes. */
+	&.spectrum-Search {
+		--spectrum-search-border-radius: 0;
+		--spectrum-search-edge-to-visual: var(--spectrum-field-edge-to-visual-quiet);
 	}
 
 	&.spectrum-Search .spectrum-Search-input {
 		border-radius: var(--mod-search-border-radius, var(--spectrum-search-border-radius));
 		padding-inline-start: calc(var(--mod-search-edge-to-visual, var(--spectrum-search-edge-to-visual)) + var(--mod-search-icon-size, var(--spectrum-search-icon-size)) + var(--mod-search-text-to-icon, var(--spectrum-search-text-to-icon)));
-		padding-inline-end: calc(var(--mod-search-button-inline-size, var(--spectrum-search-button-inline-size)) - var(--mod-search-quiet-button-offset, var(--spectrum-search-quiet-button-offset)));
+		padding-inline-end: calc(var(--mod-search-button-inline-size, var(--spectrum-search-button-inline-size)) - var(--mod-search-quiet-button-offset, calc(var(--mod-search-block-size, var(--spectrum-search-block-size)) / 2 - var(--mod-search-icon-size, var(--spectrum-search-icon-size)) / 2)));
```

Comment:
Now that the clear button is no longer sitting outside the bounds of the quiet search, can this be simplified to just 
`padding-inline-end: var(--mod-search-button-inline-size, var(--spectrum-search-button-inline-size));`? 
I noticed that the padding does not account for the full size of the button:
<img width="399" alt="search-field-padding" src="https://github.com/user-attachments/assets/c0575c5f-7958-4056-a32c-2411c6b3aab3">

Reply:
I think it's meant to be there as a guardrail to prevent text from impeding on the close icon, even if the icon sizing is altered.

Reply:
It seems odd to me from an interaction standpoint for the text to go under the clickable button. If longer text goes all the way to the end of the input (of the blue in that screenshot), then when the user may be thinking they're clicking at the end of the text to edit, they'd actually be clicking the clear button by mistake.

AFAIK design did not specify that increased padding for quiet, and React doesn't have it either. I think it may have just been always a little wrong and tied to the horizontal mis-positioning that you fixed in this PR.

Reply:
👍 This is written up in **CSS-1052**.